docs/manuals/cli/run.md

```diff
@@ -0,0 +1,118 @@
+# Run application in Taipy CLI
+
+Assume that you have a "main.py" file containing your Taipy application.
+To start the Taipy application, you can run:
+
+```console
+$ taipy run main.py
+```
+
+!!! note
+
+    An alternative way to run your application is to use the Python command directly.
+
+    ```console
+    $ python main.py
+    ```
+    However, when working with command-line arguments, Taipy arguments can be confused with arguments
+    of other libraries or your application's arguments. To avoid this confusion, we recommend
+    using the `taipy run` command, which is more robust in avoiding any command-line argument conflict.
+
+## With Taipy arguments
```

Comment:
May be you want to add xrefs to the Taipy Core, Taipy GUI and Taipy REST specific command line options help pages?

Reply:
That makes sense. Will do.